testdata/p4tc_samples_outputs/indirect_counter_01_example.template

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/indirect_counter_01_example numtables 1
+
+$TC p4template create action/indirect_counter_01_example/ingress/send_nh actid 1 \
+	param port_id type dev \
+	param dmac type macaddr \
+	param smac type macaddr
+$TC p4template update action/indirect_counter_01_example/ingress/send_nh state active
+
+$TC p4template create action/indirect_counter_01_example/ingress/drop actid 2
+$TC p4template update action/indirect_counter_01_example/ingress/drop state active
+
+$TC p4template create extern/root/Counter extid 0x19000000 numinstances 1 tc_acl 0x1136 has_exec_method 
```

Comment:
Minor nit, there's an extra whitespace

```suggestion
$TC p4template create extern/root/Counter extid 0x19000000 numinstances 1 tc_acl 0x1136 has_exec_method
```

Reply:
As the PR is merged, will it be okay if i address this in InternetChecksum PR?

Reply:
Addressed in PR https://github.com/komaljai/p4c/pull/26